src/help/zaphelp/contents/cmdline.html

```diff
@@ -31,17 +39,17 @@
 Relative paths to session file are resolved against the "session" directory located in ZAP's home directory (default or specified with -dir option).
 <br/>
 Configuration keys should be specified using the dot notation based their location in the XML of the configuration file, eg:<br/>
-<pre>-config api.key=12345 -config connection.timeoutInSecs=60</pre>  
+<pre>zap.sh -config api.key=12345 -config connection.timeoutInSecs=60</pre>  

```

Comment:
We deliberately dont use zap.sh as most of our users are on Windows and would therefore use zap.bat